mls-rs-crypto-hpke/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "mls-rs-crypto-hpke"
-version = "0.8.0"
+version = "0.8.1"
```

```suggestion
version = "0.9.0"
```
Because `mls-rs-core` is part of the public API of this crate (e.g. with `HpkeContextS`)